docs/_includes/text-mono.adoc

```diff
@@ -97,9 +97,9 @@ There are three scenarios that are affected by this syntax change.
 
 Let's first consider the legacy syntax used in each of these examples.
 
-* +monospaced normal text+ or \`monospaced normal text`
+* `monospaced normal text` or \`monospaced normal text`
 * \`Use {asciidoctor-version} to print the version of Asciidoctor` (attribute is _not_ replaced)
-* +The version of Asciidoctor is {asciidoctor-version}+ (attribute _is_ replaced)
+* `The version of Asciidoctor is {asciidoctor-version}` (attribute _is_ replaced)
```

Comment:
We don't want to replace these because they are showing how to use the legacy syntax.